std_msgs/Header header
perception_unit/DetectedObject[] objects